docs/guides/universal-receiver-delegate/accept-reject-assets.md

```diff
@@ -0,0 +1,213 @@
+
+// modules
+import {LSP1UniversalReceiverDelegateUP} from "https://github.com/lukso-network/lsp-smart-contracts/blob/v0.6.2/contracts/LSP1UniversalReceiver/LSP1UniversalReceiverDelegateUP/LSP1UniversalReceiverDelegateUP.sol";
+
+// constants
+import {_TYPEID_LSP7_TOKENSRECIPIENT} from "https://github.com/lukso-network/lsp-smart-contracts/blob/v0.6.2/contracts/LSP7DigitalAsset/LSP7Constants.sol";
+import {_TYPEID_LSP8_TOKENSRECIPIENT} from "https://github.com/lukso-network/lsp-smart-contracts/blob/v0.6.2/contracts/LSP8IdentifiableDigitalAsset/LSP8Constants.sol";
+
+contract CustomUniversalReceiverDelegate is LSP1UniversalReceiverDelegateUP  {
+
+    address immutable public owner;
+    mapping (address => bool) public allowedAssets;
+
+    constructor(address _owner){
+        owner = _owner;
+    }
+
+    modifier onlyOwner(){
+        require(msg.sender == owner, "CustomUniversalReceiverDelegate : Caller is not the owner");
+        _;
+    }
+
+
+    function setAllowedAssets(address assets) public onlyOwner {
+        allowedAssets[assets] = true;
+    }
+    /**
+    * @dev Reverts when the asset being transferred is not allowed. If allowed, the address of the asset
+    * will be registered inside the storage, and removed when balance of the asset equal 0, according to
+    * the LSP5-ReceivedAssers standard.
+    *
+    * @param caller The address of the asset informing the `universalReceiver(..)` function on the UniversalProfile.
+    * @param value The amount of native tokens sent by the caller to the universalReceiver function on the UniversalProfile.
+    * @param typeId The typeId representing the context of the call to the universalReceiver function on the UniversalProfile.
+    * @param typeId The data sent to the universalReceiver function on the UniversalProfile.
+    */
+    function universalReceiverDelegate(
+        address caller,
+        uint256 value,
+        bytes32 typeId,
+        bytes memory data
+    ) public override returns (bytes memory result){
+        // checking if the asset being transferred is allowed or not.
+        if(typeId == _TYPEID_LSP8_TOKENSRECIPIENT || typeId == _TYPEID_LSP7_TOKENSRECIPIENT){
+            require(allowedAssets[caller], "Asset being transferred is not allowed to be received");
+        }
+        // using the default implementation code to register the address of assets received
+        result = super.universalReceiverDelegate(caller, value, typeId, data);
+    }
+}
+```
+
+The code above will register the address of the assets allowed and remove them when the UP's balance for this asset is equal to 0. It will also reject assets that are not allowed. Since this code will need **SETDATA Permission**, after deploying you will set the address of the URD in the storage using the code from the **[Set the address of the URD in the storage](./set-default-implementation.md#set-the-address-of-the-urd-in-the-storage)** section.
```

Comment:
Maybe you can add a link to this page:
<https://docs.lukso.tech/standards/universal-profile/lsp6-key-manager#permissions> on "SETDATA Permission" ?